Style.css

```diff
@@ -0,0 +1,62 @@
+.tableHeader{
+	text-align:center;
+	color:#141817;
+	font-size: 50px;
+	background-color: #fff;
+	border-bottom-style:dashed;
+	border-bottom-color:#0a2e68;
+}
+
+table{
```

Comment:
It looks to me like this style is meant to apply to a specific table, but you're applying it to *all* tables. That might work right now, but it makes the code hard to maintain because you can't add any other tables later. Instead, give this specific table a class and use that to apply CSS to it.